pages/faq/troubleshooting/faq.md

```diff
@@ -1,52 +1,57 @@
+- [What is the performance impact on the gateway device due to encryption?](#what-is-the-performance-impact-on-the-gateway-device-due-to-encryption)
+- [How long does the update process run typically? For now it appears to be quick for small updates.](#how-long-does-the-update-process-run-typically-do-you-have-any-benchmark-data-for-now-it-appears-to-be-quick-for-small-updates)
+- [How does the device registration work over the VPN and how do you ensure the identity of the device on the first-time registration?](#how-does-the-device-registration-work-over-the-vpn-and-how-do-you-ensure-the-identity-of-the-device-on-the-first-time-registration)
+- [If the device is installed behind a proxy/firewall and can’t be reachable on Internet via direct connection, what are the pitfalls?](#if-the-device-is-installed-behind-a-proxyfirewall-and-cant-be-reachable-on-internet-via-direct-connection-what-are-the-pitfalls)
+- [How do you secure your own “cloud” to prevent malicious attack which may allow attacker to break-in our systems?](#how-do-you-secure-your-own-cloud-to-prevent-malicious-attack-which-may-allow-attacker-to-break-in-our-systems)
+- [What does it mean when a device type is discontinued?](#what-does-it-mean-when-a-device-type-is-discontinued)
+- [I have a device that is not on the supported devices list. Can it run on {{ $names.company.lower }}?](#i-have-a-device-that-is-not-on-the-supported-devices-list-can-it-run-on-{{ $names.company.lower }})
+- [What to keep in mind when choosing power supply units?](#what-to-keep-in-mind-when-choosing-power-supply-units)
+- [Does {{ $names.company.lower }} have access to my device, source code and images?](#does-{{ $names.company.lower }}-have-access-to-my-device-source-code-and-images)
 
 ##### Can I use multiple containers?
-Multiple container applications are supported, beginning with {{ $names.os.lower }} v2.12.0. To run multiple containers, you will need to create or upgrade to a [starter or microservices type application][app-types] and include a `docker-compose.yml` file at the root of your project. You can reference the [multicontainer documentation][multicontainer] for more details on the supported configurations.
 
-__Note:__ If you do not see an option to choose a starter or microservices application type, a multicontainer compatible OS version has not yet been released for the selected device type.
+Multiple container fleets are supported, beginning with {{ $names.os.lower }} v2.12.0. To run multiple containers, you will need to create or upgrade to a [starter or microservices type fleet][app-types] and include a `docker-compose.yml` file at the root of your project. You can reference the [multicontainer documentation][multicontainer] for more details on the supported configurations.
+
+**Note:** If you do not see an option to choose a starter or microservices fleet type, a multicontainer compatible OS version has not yet been released for the selected device type.
```

Comment:
We *must* be able to get rid of this by now, right? :)

Reply:
asked the team to be on the safe side, I have seen this line somewhere else as well 
https://www.flowdock.com/app/rulemotion/r-resinos/threads/DvTz_S6an_5lSptgm98YuCGir6n

Reply:
Alex replied here:
> From what I can tell multicontainer was introduced with supervisor 7.0.0, and the first time that it made it to meta-balena was v2.11.0 with SV v7.1.0. All DTs have released above 2.11 afaik, so yes.